package/boot/uboot-rockchip/patches/004-add_rock64_gen3_support.patch

```diff
@@ -0,0 +1,84 @@
+Use the same approach as ROC-RK3328-CC which enables SPL GPIO,
```

Comment:
I would prefer to track the complete Git commit header from upstream commit `33863f744d513f5c16a254870e7b3cef8580bbc9` here